rio/src/RIO/Prelude/Logger.hs

```diff
@@ -596,19 +597,32 @@ stickyImpl ref lo logFunc loc src level msgOrig = modifyMVar_ ref $ \sticky -> d
     LevelOther "sticky-done" -> do
       clear
       logFunc loc src LevelInfo msgOrig
-      return mempty
+      return (mempty,0)
     LevelOther "sticky" -> do
       clear
       let bs = toStrictBytes $ toLazyByteString $ getUtf8Builder msgOrig
       logSend lo (byteString bs <> flush)
-      return bs
+      return (bs, utf8CharacterCount bs)
     _
       | level >= logLevel -> do
           clear
           logFunc loc src level msgOrig
           unless (B.null sticky) $ logSend lo (byteString sticky <> flush)
-          return sticky
-      | otherwise -> return sticky
+          return (sticky,stickyLen)
+      | otherwise -> return (sticky,stickyLen)
+
+-- | The number of Unicode characters in a UTF-8 encoded byte string.
+utf8CharacterCount :: ByteString -> Int
+utf8CharacterCount = go 0
+  where
+    go n bs = case B.uncons bs of
```

Comment:
I think the `n` should have a bang pattern on it to avoid accumulating thunks.

Reply:
🤦‍♂ Of course, silly me..